src/components/termsOfService/TermsOfService.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Container from '../common/Container';
+import { StyledTermsOfService, StyledTermsOfServiceContainer } from './style';
+
+const TermsOfService = () => (
+  <Container>
+    <StyledTermsOfService>
+      <StyledTermsOfServiceContainer>
+        <h2> TERMS OF SERVICE FOR SAY THEIR NAMES</h2>
+        <p>
+          By accessing the website at
+          {' '}
+          <Link to="/">https://saytheirnames.io</Link>
```

Comment:
```suggestion
          <Link to="/">saytheirnames.io</Link>
```
To keep consistent with style in the Privacy Policy and the rest of the website.